train.py

```diff
@@ -218,14 +218,10 @@ def freeze_backbone(m):
                     progress_bar.update()
                     continue
                 try:
-                    imgs = data['img']
-                    annot = data['annot']
-
-                    if params.num_gpus == 1:
-                        # if only one gpu, just send it to cuda:0
-                        # elif multiple gpus, send it to multiple gpus in CustomDataParallel, not here
-                        imgs = imgs.cuda()
-                        annot = annot.cuda()
+                    # if only one gpu, just send it to cuda:0
```

Comment:
in fact, this is also on purpose as it was said on comments.
What you do is to move all data to cuda0, then it will cause malloc imbalance.